### PR TITLE
Allow users with write permission to run actions

### PR DESCRIPTION
**Pull request by NishiyamaPedro**
---
I have a use case where I need a team to be able to run actions without admin access.

### CODE REVIEW

File: routers/web/repo/actions/actions.go [hunk 1]
```diff
@@ -168,7 +168,7 @@ func List(ctx *context.Context) {
 	actionsConfig := ctx.Repo.Repository.MustGetUnit(ctx, unit.TypeActions).ActionsConfig()
 	ctx.Data["ActionsConfig"] = actionsConfig
 
-	if len(workflowID) > 0 && ctx.Repo.IsAdmin() {
+	if len(workflowID) > 0 && ctx.Repo.CanWrite(unit.TypeActions) {
 		ctx.Data["AllowDisableOrEnableWorkflow"] = true
```
Comment (Zettat123) [inline routers/web/repo/actions/actions.go]: I agree with the idea that users with write access can manually run workflows. But I think only repo admins should be able to disable or enable workflows.

**Comment by Zettat123** [comment]
According to the [GitHub documentation](https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow#configuring-a-workflow-to-run-manually), users with write permissions can run workflows manually. I think we can support this in Gitea.

File: routers/web/repo/actions/actions.go (hunk 1 above)
Comment (NishiyamaPedro) [reply routers/web/repo/actions/actions.go]: You're right. Fixed.

**Review by Zettat123 — APPROVED** [review]

**Comment by yp05327** [comment]
I prefer to add a test for this.

**Review by wxiaoguang — APPROVED** [review]
Seems to be simple enough

**Pull request merged**